.github/workflows/main.yml

```diff
@@ -5,6 +5,11 @@ on:
   push:
     branches:
```

Comment:
Currently beachball will recognize changes to these files and require change files to be made. If that doesn't run on the CI then those change files will not get cleaned correctly.

- Update the beachball configuration to ignore these files: https://microsoft.github.io/beachball/overview/configuration.html#options
- This should be something you can manually test locally is being properly ignored

Reply:
Added exclusions to Beachball config. Through manual testing, I found it needed slightly different patterns.